scripts/section_data.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "learning_path_group": "Introduction",
+    "dirName": "introduction",
+    "workbook": "01-introduction.asciidoc",
+    "translations": ["de", "it", "ja", "zh", "ru"],
+    "image": "images/learn/LP_thumbnail_introduction.jpg",
```

Comment:
Are these files in the other repo?  I feel like they should be here so as to remove this coupling.

Reply:
Most of the time the thumbnail is retrieved from YouTube (see https://github.com/InnerSourceCommons/innersourcecommons.net/blob/master/content/learn/learning-path/contributor/01.md for example) - this is just a backup for where there is no video, which is only the workbooks.

I do think we need to choose an image for workbooks, as using intro thumbnails isn't really relevant. I tried rendering the site without images for workbooks but it doesn't really work.

Can we raise an issue for a workbook image separately, and address the coupling at the same time?

Reply:
https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/375

Reply:
It seems like the way to resolve the coupling issue would be to move the images to the InnerSourceLearningPath repo and then copy them over to innersourceommmons.org along with the articles - no?